classes/form/upload_bookings_form.php

```diff
@@ -0,0 +1,73 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace mod_facetoface\form;
+
+use moodle_url;
+use html_writer;
+
+/**
+ * Upload bookings form class
+ *
+ * @package    mod_facetoface
+ * @author     Kevin Pham <kevinpham@catalyst-au.net>
+ * @copyright  Catalyst IT, 2024
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class upload_bookings_form extends \moodleform {
+
+    /**
+     * Build form for importing woekflows.
+     *
+     * {@inheritDoc}
+     * @see \moodleform::definition()
+     */
+    public function definition() {
+        global $CFG;
+
+        $mform = $this->_form;
+
+        $mform->addElement('header', 'settingsheader', get_string('upload'));
+
+        $url = new moodle_url('example.csv');
```

Comment:
Not sure if there is a convention or code style guide on this, but generally im partial to absolute URLs where we can, so the PHP file/function can be moved around without invisibly breaking a link to a static asset

Reply:
I blatantly copied the style from here:
https://github.com/moodle/moodle/blob/master/admin/tool/uploaduser/user_form.php#L43

But I can change it to absolute if preferred.